README.md

```diff
@@ -416,13 +416,15 @@ public static void main(java.lang.String[])
   - **OPERANDS** shows the stacked items on memory.
   - **LOCAL STORAGE** shows the stacked items on a method.
 
-## PHP problems
-- **Problem 1:** PHP cannot calculate big numbers because PHP is different from Java.
-  But PHPJava uses `bcmath` functions and `gmp` functions to a certain extent to cover such calculations.
-  Therefore, PHPJava returns a mixed value and we recommend to cast them to `string` on PHPJava.
-
-- **Problem 2:** PHPJava cannot completely cover Java types because PHP is different from Java.
-  The comparison table of Java and PHPJava is shown below:
+## About big number calculation
+- In normally, PHP cannot calculate big numbers as such as `long` and `double` types.
+  But, PHPJava uses external `Math` library for covering above problems.
+  And, PHPJava to use Java's type as below comparison table.
+  Therefore, we recommend to cast them to `string` on PHPJava.
+  And, if it can be calculated with 64-bitPHP, PHPJava uses PHP's arithmetic operations.
+  
```

Comment:
Normally, PHP cannot calculate big numbers such as `long` and `double` types but PHPJava uses an external `Math` library to cover this issue as shown in the comparison table below. For this reason, we recommend you to cast those types to `string` in PHPJava. Additionally, if it . . .